docs/source/terms_and_model.rst

```diff
@@ -342,11 +342,120 @@ Location for Variation.
 **Implementation Guidance**
 
 * Location refers to a position.  Although it MAY imply a sequence,
-  the two concepts are not interchangable, especially when the
+  the two concepts are not interchangeable, especially when the
   location is non-specific (e.g., a range) or symbolic (a gene).
```

Comment:
The document states ` Two objects are considered equal if and only if their respective attributes are equal.`.  In the context of ChromosomeLocation, should the document provide guidance / examples  for equality, contains, or other operations?

Reply:
I personally think that this is already covered for identity with the excerpt you provided, and that more complex comparisons are out of scope.

Would you like to propose text for consideration?

Reply:
Does MIchael's comment above https://github.com/ga4gh/vr-spec/pull/203#discussion_r448606281 apply?  I've looked for a succinct write up of karyotype / chromosome parsing and operations we might link to, but have been unable to find one.

Reply:
@bwalsh Not sure what/how to answer this, but let's try:

* any cytogenetic band annotation is _unambiguous_ in its meaning; there are no alternative annotations, only higher or lower hierarchical expressions; therefore, cytogenetic "positions" ("bands" in the widest sense up to chromosome level) can be expressed through a tree structure
* any single cytogenetic band represents a "genomic interval", i.e. can be expressed as an instance of a _SimpleInterval_ if mapped to a reference genome using some coordinate system, depending on a given reference genome
  - most commonly using an ~800-900 band mapping, e. g. provided by UCSC
* cytobands cannot be represented as single precise genome positions; genome positions can be referred to by cytobands (lossy)
* cytobands are easy; karyotype annotations aren't

Btw., I'm trying to track these developments with a service - so please settle on a syntax :-)

* [progenetix.org/services/cytomapper?cytoBands=8q21q24.1&assemblyId=GRCh38](https://progenetix.org/services/cytomapper?cytoBands=8q21q24.1&assemblyId=GRCh38)
* [progenetix.org/services/cytomapper/?type=ChromosomeLocation&assemblyId=hg18&chr=17&start=p11&end=q21](https://progenetix.org/services/cytomapper/?type=ChromosomeLocation&assemblyId=hg18&chr=17&start=p11&end=q21)
* [progenetix.org/services/cytomapper/?assemblyId=GRCh38&chroBases=17:800000-24326000](https://progenetix.org/services/cytomapper/?assemblyId=GRCh38&chroBases=17:800000-24326000)